glib/src/gobject/auto/binding_group.rs

```diff
@@ -0,0 +1,124 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::object::IsA;
+use crate::object::ObjectType as ObjectType_;
+use crate::signal::connect_raw;
+use crate::signal::SignalHandlerId;
+use crate::translate::*;
+use crate::BindingFlags;
+use crate::Object;
+use crate::StaticType;
+use std::boxed::Box as Box_;
+use std::fmt;
+use std::mem::transmute;
+
+crate::wrapper! {
+    #[doc(alias = "GBindingGroup")]
+    pub struct BindingGroup(Object<gobject_ffi::GBindingGroup>);
+
+    match fn {
+        type_ => || gobject_ffi::g_binding_group_get_type(),
+    }
+}
+
+impl BindingGroup {
```

Comment:
Should probably manually implement like the current GBinding?

Reply:
Yes, jut realized I forgot that. At the very least `bind` needs it

Reply:
Copied most of the implementation from `BindingBuilder`